config/config.exs

```diff
@@ -21,6 +21,10 @@ use Mix.Config
 #     config :logger, level: :info
 #
 
+config :logger, :console,
+  format: "\n$time $metadata [$level] $message",
+  metadata: [:service_name]
```

Comment:
Same question about BM's config.  Will the config merge or is it up to BM to add `[:service_name]`

Reply:
I'll need to research how "third party configs" work in Elixir. Great concern....I'll do some tests and get back to ya...

Reply:
![image](https://user-images.githubusercontent.com/6567505/86834344-f991ef80-c057-11ea-920f-5a57e80712d0.png)

Reply:
I.e. I don't think the config file from the dependency matters unless you actually run `iex` within that module.

Reply:
To answer the second question: BM will need to add `[:service_name]` to the metadata config.

Reply:
🙇